.github/workflows/readme-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Check README Updates
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
```

Comment:
I think we must change this like below to have the action picked up and ran:
```
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  workflow_dispatch:
  merge_group:
```

Reply:
Sorry, I updated the comment in the meantime, please check again. This [5959247](https://github.com/paritytech/polkadot-sdk/pull/6359/commits/5959247bf0c3a0b3f8b21f8d255c1c184d69659c) is not complete.

Reply:
Yes I did see the update in the comment and pushed the change then